.github/workflows/ci_cd.yml

```diff
@@ -115,9 +115,9 @@ jobs:
         run: "license-checker --onlyAllow '0BDS;MIT;Apache-2.0;ISC;BSD-3-Clause;BSD-2-Clause;CC-BY-4.0;CC-BY-3.0;BSD;CC0-1.0;Unlicense;UNLICENSED' --summary"
 
   publish:
-    name: Publish Capabilities to Superface.ai
+    name: Publish to Superface.ai
     runs-on: ubuntu-latest
-    needs: ['test', 'test_capabilities', 'license_check']
+    needs: ['test', 'license_check']
```

Comment:
In order for this step to be transformed 1:1, this would also depend on `test_tools`. Just to be clear, was this removed because `test` is already dependant on `test_tools` and including it here would be redundant?

Reply:
It was intentional, but I don't know if it was good idea, becuase now linting for scripts and tests will run in parallel.

<img width="946" alt="image" src="https://user-images.githubusercontent.com/959390/143075619-0972882c-2baa-4ec7-adc3-f13648270687.png">

Reply:
I renamed stages to have clearer name (I hope) and also allowed to run tools tests and Station test in parallel.

<img width="966" alt="image" src="https://user-images.githubusercontent.com/959390/143224494-0bcc3d74-1bc4-421c-96c6-43b1ee925f9b.png">

Reply:
Yes, that's nice 👍